bindata/assets/alerts/kube-apiserver-slos.yaml

```diff
@@ -71,18 +71,18 @@ spec:
         # resource-scoped latency
         label_replace(
           (
-            sum(rate(apiserver_request_duration_seconds_count{job="apiserver",verb=~"LIST|GET",subresource!~"proxy|log|exec",scope="resource"}[5m]))
```

Comment:
why no portforward here?

Reply:
does `portforward`  uses GET or LIST?

Reply:
Attach and port-forward and exec look the same in k8s.io/kubernetes/pkg/registry/core/pod/rest/subresources.go. So I would expect something uniform here.

Reply:
ack